pkg/policy/groups/aws/aws.go

```diff
@@ -51,7 +51,15 @@ func GetIPsFromGroup(ctx context.Context, group *api.ToGroups) ([]net.IP, error)
 // getInstancesFromFilter returns the instances IPs in aws EC2 filter by the
 // given filter
 func getInstancesIpsFromFilter(ctx context.Context, filter *api.AWSGroup) ([]net.IP, error) {
+	result := []ec2_types.Reservation{}
```

Comment:
same here for the initialization of the slice.

Reply:
Updated